client/lib/plans/features-list.js

```diff
@@ -615,6 +609,26 @@ export const FEATURES_LIST = {
 			i18n.translate( 'Live chat support to help you get started with your site.' ),
 	},
 
+	[ constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS ]: {
+		getSlug: () => constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
+		getTitle: () => i18n.translate( 'Email & Live Chat Support' ),
+		getDescription: () =>
+			i18n.translate(
+				'Live chat is available 24 hours a day from Monday through Friday. ' +
+					'You can also email us any day of the week for personalized support.'
+			),
+	},
+
+	[ constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_ALL_DAYS ]: {
+		getSlug: () => constants.FEATURE_EMAIL_LIVE_CHAT_SUPPORT_ALL_DAYS,
+		getTitle: () => i18n.translate( 'Email & Live Chat Support' ),
+		getDescription: () =>
+			i18n.translate(
+				'Live chat is available 24/7. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Live chat is available 24/7. You can also email us any day of the week for personalized support for your site.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/plan-page-tooltip-copy)